### PR TITLE
update to latest slack-api-ref

### PR DESCRIPTION
**Pull request by jmanian**
There's a potentially breaking change in here, which is that they seem to have removed `groups.close`. I'm not sure if there's a special way to call that out on the changelog (I bolded it), or if that has implications for versioning (0.10?).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ### 0.9.2 (Next)
 
+* [#169](https://github.com/slack-ruby/slack-ruby-client/pull/169): Updated to latest slack-api-ref; adds [Conversations API](https://api.slack.com/docs/conversations-api) and support for locale; **removes `groups_close`** - [@jmanian](https://github.com/jmanian).
```
Comment (dblock) [inline CHANGELOG.md]: We usually just call these things as separate line items, no need to bold anything.

So 

```
[#169](https://github.com/slack-ruby/slack-ruby-client/pull/169): Added [Conversations API](https://api.slack.com/docs/conversations-api) - [@jmanian](https://github.com/jmanian).
[#169](https://github.com/slack-ruby/slack-ruby-client/pull/169): Removed `groups_close` - [@jmanian](https://github.com/jmanian).
```

File: bin/commands/channels.rb [hunk 2]
```diff
@@ -38,6 +38,7 @@
   g.long_desc %( Gets information about a channel. )
   g.command 'info' do |c|
     c.flag 'channel', desc: 'Channel to get info on.'
+    c.flag 'include_locale', desc: 'Set this to true to receive the locale for this channel. Defaults to false.'
```
Comment (dblock) [inline bin/commands/channels.rb]: This is new, so needs a changelog line too.

Comment (jmanian) [reply bin/commands/channels.rb]: Yep, it's hidden in the middle of that one long changelog item I wrote 😉 . I'll split it up into 3 items.

**Comment by dblock** [comment]
This will need to bump version to 0.10.0 indeed. Want to help out and make the next release @jmanian?

**Comment by jmanian** [comment]
Sure, I'll give it a shot. Does this give you what you need https://rubygems.org/profiles/jmanian

**Comment by dblock** [comment]
Sent you an [invite](https://github.com/slack-ruby/slack-ruby-client/invitations) and added to Rubygems, assigning #170 to you. Please follow https://github.com/slack-ruby/slack-ruby-client/blob/master/RELEASING.md

**Comment by jmanian** [comment]
Oh, did you want me to do the 0.10.0 bump as part of this PR?

**Comment by dblock** [comment]
You can do that part of the release, no need to PR.

**Pull request merged**